packages/trader/src/sass/app/modules/mt5/mt5-dashboard.scss

```diff
@@ -95,6 +95,11 @@
             width: 40%;
```

Comment:
nit: we can use `include desktop { .. }` for desktop only values instead of unsetting the values in `mobile`.

Reply:
ok, good point 👍